### ISSUE TITLE
version upgrade to glpi 9.5

### ISSUE THREAD

**Issue by ghost** [Turgon37/docker-glpi#50]
I have installed the image `turgon37/glpi:nginx-72-latest`
How can I upgrade from version to glpi 9.5 ?

**Comment by Turgon37** [comment]
Hello,

My git repository only build docker image on "releases", the 9.5 is currently only a release candidate, not a final release.
To upgrade to 9.5 you have to wait the 9.5 will be officially available and pull the new docker image once it will be available.
I suggest you to set the glpi version in image name when you pull the image in your infrastructure, I provide the tag nginx-72-9.4.6-latest in addition of the nginx-72-latest. It prevent unwanted upgrades

**Comment by gatno** [comment]
Hello @Turgon37 , 

now the GLPI 9.5 stable is available.

Regards

**Comment by Turgon37** [comment]
Hello I'm working on 9.5.0 version because it requires some updates in my build configuration

**Comment by Turgon37** [comment]
The new image is available here turgon37/glpi:nginx-72-9.5.0-latest Please take care of the indications shown by GLPI web page during the upgrade step. You have to execute some manual actions like

```
docker-compose exec -u www-data glpi bin/console glpi:migration:timestamps
```
